charts/snowplow-stream-collector/README.md

```diff
@@ -0,0 +1,246 @@
+
+_Note_: The HealthCheck you create should map to the same port you used for the service deployment.
+
+#### Target: PubSub
+
+To send data into PubSub you will need to bind a valid GCP service-account to the service deployment.  In Terraform this looks something like the following:
+
+```hcl
+resource "google_service_account" "snowplow_stream_collector" {
+  account_id   = "snowplow-stream-collector"
+  display_name = "Snowplow Stream Collector service account"
+}
+
+resource "google_service_account_iam_binding" "snowplow_stream_collector_sa_wiu_binding" {
+  role = "roles/iam.workloadIdentityUser"
+  members = [
+    "serviceAccount:<project>.svc.id.goog[default/snowplow-stream-collector]"
+  ]
+  service_account_id = google_service_account.snowplow_stream_collector.id
+}
+
+resource "google_project_iam_member" "snowplow_stream_collector_sa_pubsub_viewer" {
+  role   = "roles/pubsub.viewer"
+  member = "serviceAccount:${google_service_account.snowplow_stream_collector.email}"
+}
+
+resource "google_project_iam_member" "snowplow_stream_collector_sa_pubsub_publisher" {
+  role   = "roles/pubsub.publisher"
+  member = "serviceAccount:${google_service_account.snowplow_stream_collector.email}"
+}
+
+output "snowplow_stream_collector_sa_account_email" {
+  value = google_service_account.snowplow_stream_collector.email
+}
+```
+
+You can then use the resulting value as an input to `serviceAccount.gcp.serviceAccount` which will allow the deployment to access PubSub.
+
+You will need to fill these targeted fields:
+
+- `cloud: "gcp"`
+- `serviceAccount.deploy: true`
+- `serviceAccount.gcp.serviceAccount: <output_value_from_above>`
+
+### AWS (EKS) settings
+
+#### TargetGroup binding
+
+To manage the load balancer externally to the kubernetes cluster you can bind the deployment to an existing TargetGroup ARN.  Its important that the TargetGroup exist ahead of time and that you use the same port as you have used in your `values.yaml`. 
+
+_Note_: Before this will work you will need to install the `aws-load-balancer-controller-crds` and `aws-load-balancer-controller` charts into your EKS cluster.
+
+You will need to fill these targeted fields:
+
+- `cloud: "aws"`
+- `service.aws.targetGroupARN: "<target_group_arn>"`
+
+#### Target: Kinesis and/or SQS
+
+To send data into Kinesis and/or SQS without hardcoded credentials you will need to bind a valid AWS IAM role ARN to the service deployment.  In Terraform this looks something like the following:
+
+```hcl
+resource "aws_iam_policy" "snowplow_stream_collector" {
+  name = "snowplow-stream-collector"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "kinesis:DescribeStream",
+        "kinesis:DescribeStreamSummary",
+        "kinesis:List*",
+        "kinesis:Put*",
+        "sqs:GetQueueUrl",
+        "sqs:SendMessage",
+        "sqs:ListQueues"
+      ],
+      "Resource": "*",
```

Comment:
I would suggest mentioning that best security policy is to restrict the resource down to just what you need. For example, if your kinesis streams were called `good` and `bad` and your AWS account id was 333221111111, then you would do:

```
            "arn:aws:kinesis:us-east-1:333221111111:stream/bad",
            "arn:aws:kinesis:us-east-1:333221111111:stream/good"
```

Reply:
Yeah thats a fair point - I have done this in the [OS Terraform Modules](https://github.com/snowplow-devops/terraform-aws-collector-kinesis-ec2/blob/main/main.tf#L105-L115) - will replicate it here.